test/verify/check-pages

```diff
@@ -956,7 +952,7 @@ OnCalendar=daily
 
         # Upload permission error
         b.drop_superuser()
-        b.set_file_autocomplete_val("#demo-upload", "/root/")
+        b.set_file_autocomplete_val("#demo-upload", "/var/")
```

Comment:
Why the path change?

Reply:
/root is not listable by admin, and our FileAutocomplete widget shows an error and doesn't let us select /root. Previously, it was possible to make any value stick as the selected value just by closing the dropdown (regardless of whether it exists, or an error, occured, even for `!isCreatable`). I consider that a bug, actually. 

Now this is no longer possible and we need to use something that can be selected from the dropdown, but not be written to by admin. Also, we need to use something that is not a read-only filesystem on CoreOs.

So there are bugs here still to be fixed, I guess:

 - FileAutocomplete should probably not prevent the user from selecting a directory that can't be listed.
 - The upload machinery or the test here don't react correctly to a read-only filesystem, only to read-only permission bits.